elements/pl-hidden-hints/pl-hidden-hints.py

```diff
@@ -16,7 +17,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     pl.check_attribs(element, [], [])
 
     # Parse hints from frontend
-    hints: List[Tuple[int, int, Optional[str], str]] = []
+    hints: list[tuple[int, int, Optional[str], str]] = []
```

Comment:
Are we going to switch entirely to lowercase types? IMO we should do that all in one go for consistency.

Reply:
I can bundle that here. There aren't a ton of uses.